packages/regl-worldview/src/WorldviewContext.js

```diff
@@ -207,6 +209,18 @@ export class WorldviewContext {
   };
 
   paint() {
+    try {
+      this._paint();
+    } catch (error) {
+      if (error.message === "(regl) context lost") {
```

Comment:
any chance you could persist the comment added in `Worldview.js` so we keep the context?